src/Microsoft.Data.SqlClient/netcore/src/Microsoft.Data.SqlClient.csproj

```diff
@@ -918,11 +918,14 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Strings.resx</DependentUpon>
     </Compile>
+    <Compile Include="Microsoft\Data\SqlClientX\Handlers\DataSourceHandler.cs" />
```

Comment:
* This file doesn't exist
* IO\ITdsStream should be added to the project

Reply:
My bad. I remember having cleaned this up and I see I have a commit sitting in my local branch.